### DISCUSSION TITLE
`Default theme` documentation links to wrong Github page

### DISCUSSION THREAD

**Discussion by jpenna** [chakra-ui/chakra-ui#8864] [General] [answered]
### Description

The link for the Github package is sending to an 404 page: https://v2.chakra-ui.com/docs/styled-system/theme

> When opening the issue, there was a topic for "Documentation", but the link points to a archived repository, so I opened it here. You should probably remove the "Documentation" topic from the issues menu.

### Link to Reproduction

N/A

### Steps to reproduce

1. Go to [Default Theme](https://v2.chakra-ui.com/docs/styled-system/theme)
2. Click on [Github Source](https://github.com/chakra-ui/chakra-ui/tree/v2/packages/components/src/theme)
3. See 404 error

### Chakra UI Version

latest

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Additional Information

`ComponentLinksProps` ([file](https://github.com/chakra-ui/chakra-ui/blob/2403e843ae5364d093da8787cd28e9546bd9a4d0/website/src/components/mdx-components/component-links.tsx#L20)) is used to generate the link, but it adds a path that isn't correct for the Theme. I was going to send a PR, but I'd have to understand all the implications of changing this function and it would take longer than I'm willing to invest in this case.

**Comment by segunadebayo** [comment, marked as answer]
v2 docs are frozen. We are not shipping new v2 work.
